src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -968,6 +986,16 @@ public function setCache(array $options): object
             $this->setImmutable((bool) $options['immutable']);
         }
 
+        foreach (['proxy_revalidate', 'must_revalidate', 'no_cache', 'no_store', 'no_transform'] as $directive) {
```

Comment:
what's this list about?

Reply:
cases the are not handled above, these directives don't have values so they are either set or not.

Reply:
then the const above should describe this instead of inlining this line here
directives could be used as keys, and values would be booleans

Reply:
>directives could be used as keys 

all directives are passed to `setCache` as keys not values.
the value is either an `int` representing "seconds" or `boolean` describing whether or not to use the cache control directive.

`immutable` is handled as : 
```
if (isset($options['immutable'])) {
  $this->setImmutable((bool) $options['immutable']);
}
```

where `setImmutable` does exactly what we are doing here in the `foreach` loop.

Reply:
I was talking about the value of the const.

Reply:
you mean : 
```
cosnt NAME = [
  'foo' => false,
  'bar' => true
]
```
( where true indicates that it has a value ? )

Reply:
yes!